# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: 49da6b99bde3
# Full Hash: 49da6b99bde3c91e9ee1fe5e6d730a944c2e3890
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2020-06-10 04:36:07
# Regressor Bug: 1630434
# File Overlap Count: 1
# Description:
#   Bug 1630434 - de-duplicate preloaded intermediates that may have been cached in cert9.db r=kjacobs,bbeurdouche
#   
#   In general, PSM caches intermediates from verified certificate chains in the
#   NSS certdb. Before bug 1619021, this would include preloaded intermediates,
#   which is unnecessary because cert_storage has a copy of those certificates, and
# ==============================================================================

diff -r 882f75c5d3f9 -r 49da6b99bde3 modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Tue Jun 09 18:35:24 2020 +0000
+++ b/modules/libpref/init/all.js	Tue Jun 09 18:02:52 2020 +0000
@@ -194,9 +194,12 @@
 // Intermediate CA Preloading settings
 #if defined(MOZ_NEW_CERT_STORAGE) && !defined(MOZ_WIDGET_ANDROID)
   pref("security.remote_settings.intermediates.enabled", true);
+  pref("security.intermediate_preloading_healer.enabled", true);
 #else
   pref("security.remote_settings.intermediates.enabled", false);
+  pref("security.intermediate_preloading_healer.enabled", false);
 #endif
+pref("security.intermediate_preloading_healer.timer_interval_ms", 300000);
 pref("security.remote_settings.intermediates.bucket", "security-state");
 pref("security.remote_settings.intermediates.collection", "intermediates");
 pref("security.remote_settings.intermediates.checked", 0);